ee/cis/win-10/cis-policy-queries-WIP.yml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Allow Print Spooler to accept client connections' is set to 'Disabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting controls whether the Print Spooler service will accept client connections.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to Disabled:
+    'Computer Configuration\Policies\Administrative Templates\Printers\Allow Print Spooler to accept client connections'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\Software\\Policies\\Microsoft\\Windows NT\\Printers\RegisterSpoolerRemoteRpcEndPoint' AND data = 2);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_18.6.1
+  contributors: rachelelysia
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Point and Print Restrictions: When installing drivers for a new connection' is set to 'Enabled: Show warning and elevation prompt'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting controls whether computers will show a warning and a security elevation prompt when users create a new printer connection using Point and Print.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to 'Enabled: Show warning and elevation prompt':
+    'Computer Configuration\Policies\Administrative Templates\Printers\Point and Print Restrictions: When installing drivers for a new connection'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\Software\\Policies\\Microsoft\\Windows NT\\Printers\PointAndPrint\\NoWarningNoElevationOnInstall' AND data = 0);
```

Comment:
is the value of 'Enabled: Show warning and elevation prompt' equal to 0? I thought that 0 is mainly used for disable state

Reply:
just tried this and verified that is 0. This is surprising to me. Marking it as resolved!